docs/manual-setup.adoc

```diff
@@ -215,6 +219,16 @@ Confirm all of the transactions and verify that the TBTC has left your wallet. Y
   (top right -> Settings -> Advanced -> Reset Account)
 * During the keep-ecdsa phase of running install.sh it may complain about sha3.
   If that happens, cd into keep-ecdsa and run: `$ npm install --save-dev sha3`
+* If you see an error coming from npm complaining about dependencies (e.g. babel-cli),
+  try deleting package-lock.json (e.g. if npm cannot find file
+  local-setup/tbtc/solidity/node_modules/.staging/@keep-network/keep-ecdsa-64660964/node_modules/@babel/cli,
+  delete package-lock.json file located in local-setup/tbtc/solidity) and run
+  the script again.
+* If you cannot execute commands with bitcoind-wallet, make sure you switched to
+  version 15.8.0 of nvm.
+* If you pressed "Deposit" and got Unhandled Rejection error from dApp, make sure
+  the wallet is connected to your local ethereum network (in metamask go to
+  My Accounts->Settings->Networks, Chain Id for Localhost:8545 should be 1101).
```

Comment:
Nitpick but can we do `localhost:8545` instead of `Localhost:8545`?

Reply:
I changed it here:
https://github.com/keep-network/local-setup/pull/68